Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -1926,7 +1926,7 @@ void WebsiteDataStore::propagateSettingUpdates()
 
 #if HAVE(ALLOW_ONLY_PARTITIONED_COOKIES)
     enabled = isOptInCookiePartitioningEnabled();
-    if (m_isOptInCookiePartitioningEnabled != enabled) {
+    if (m_isOptInCookiePartitioningEnabled != enabled && trackingPreventionEnabled()) {
```

Comment:
Don’t you want to add a similar check in the network process? Otherwise it seems like network process value could become out of sync when `setOptInCookiePartitioningEnabled` is called by the NetworkSession constructor.

Reply:
Good idea